source/_docs/guides/build-tools/03-create-pr.md

```diff
@@ -0,0 +1,88 @@
+
+
+<div class="panel panel-drop panel-guide" id="accordion">
+  <div class="panel-heading panel-drop-heading">
+     <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#understand-cd"><h3 class="panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-lightbulb"></span> Continuous Delivery</h3></a>
+   </div>
+   <div id="understand-cd" class="collapse">
+     <div class="panel-inner" markdown="1">
+     Continuous delivery requires a consistently clear deployment pipeline from development to production. That is to say, an application must be able to deploy code to production at any given time regardless of current work in progress. Anything that keeps your application from deploying code to production is considered a blocker.
+
+     Production code is tracked by the master branch on GitHub and it is assumed to be production ready. Development work is done on a feature branch first, then proposed to master in the form of a Pull Request so it can be tested and reviewed before it's accepted.
+    </div>
+   </div>
+ </div>
+
+
+1. From your GitHub project page, click on the `config` directory. Select the file named `system.site.yml` and click <span class="glyphicon glyphicon-pencil"></span> to open an editor:
+
+    ![system.site.yml Configuration](/source/docs/assets/images/pr-workflow/system-site-config.png)
+
+2. Change the slogan to something inspiring:
+
+    ![Edit slogan](/source/docs/assets/images/pr-workflow/edit-slogan.png)
+
+3. Scroll down and enter a message describing this change in the **Commit changes** area. Then, click on the radio button to create a **new branch** and give it a short name, like `slogan`, then click **Propose file change**:
+
+    ![Create slogan branch](/source/docs/assets/images/pr-workflow/create-slogan-branch.png)
+
+    <div class="panel panel-drop panel-guide" id="accordion">
+      <div class="panel-heading panel-drop-heading">
+         <a class="accordion-toggle panel-drop-title collapsed" data-toggle="collapse" data-parent="#accordion" data-proofer-ignore data-target="#understand-branch-names"><h3 class="panel-title panel-drop-title" style="cursor:pointer;"><span style="line-height:.9" class="glyphicons glyphicons-lightbulb"></span> Branch Naming Conventions</h3></a>
+       </div>
+       <div id="understand-branch-names" class="collapse">
+         <div class="panel-inner" markdown="1">
+         Be sure to use unique branch names for each feature branch; the Multidev environment created on Pantheon is named after your branch. Since there is a limit to the number of characters that may be used in a Pantheon Multidev name, your environments may conflict if you always use the branch name that GitHub suggests. For details, see [Multidev](/docs/multidev/#what-are-the-naming-conventions-for-branches).
```

Comment:
This has caused problems for some folks; we are moving towards implementing a new scheme where the multidev environment is named after the pull request number.